project_task_materials/__openerp__.py

```diff
@@ -1,43 +1,22 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2012 - 2013 Daniel Reis
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     'name': 'Project Task Materials',
     'summary': 'Record products spent in a Task',
     'version': '1.0',
     'category': "Project Management",
-    'author': "Daniel Reis,Odoo Community Association (OCA)",
+    'author': 'Daniel Reis, '
+              'Antiun Ingeniería S.L., '
+              'Odoo Community Association (OCA)',
     'license': 'AGPL-3',
-    'description': """\
-Project Tasks allow to record time spent, but some activities, such as
-Field Service, often require you to keep a record of the materials spent.
-
-This module adds the ability to also this material spending.
-To use it, make sure the "Log work activities on tasks" Project setting is
-activated.
-
-Note that only a simple record is made and no accounting or stock moves are
-actually performed.""",
-    'depends': ['project', 'product'],
+    'description': "",
+    'depends': ['project', 'stock', 'stock_account', 'analytic'],
```

Comment:
We have a problem here: we can't add the `stock` additional dependencies here.
It will force people that don't use it (+accounting) to have it installed.

You should propose this as a separate module.
I suggest `project_task_materials_stock`.
You can make it `auto_install` if both `project`and `stock`are installed.

Feel free to at the same time propose changes to `project_task_materials` that make things simpler for the new module to extend it.